AutomatedTesting/Gem/PythonTests/assetpipeline/scene_tests/scene_tests.py

```diff
@@ -0,0 +1,185 @@
+import ly_test_tools.o3de.pipeline_utils as utils
+
+logger = logging.getLogger(__name__)
+targetProjects = ["AutomatedTesting"]
+
+
+# Helper: Gets a case correct version of the cache folder
+def get_cache_folder(asset_processor):
+    # Make sure the folder being checked is fully lowercase.
+    # Leave the "c" in Cache uppercase.
+    return re.sub("ache[/\\\\](.*)", lambda m: m.group().lower(), asset_processor.project_test_cache_folder())
+
+
+@pytest.fixture
+def local_resources(request, workspace, ap_setup_fixture):
+    ap_setup_fixture["tests_dir"] = os.path.dirname(os.path.realpath(__file__))
+
+
+# Set the path of the directory containing the tests, then parametrize the tests using the test_builder
+dir_test_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "tests")
+blackbox_scene_tests, blackbox_test_ids = test_builder.parametrize_blackbox_scene_test(dir_test_path)
+
+
+@pytest.mark.usefixtures("asset_processor")
+@pytest.mark.usefixtures("ap_setup_fixture")
+@pytest.mark.usefixtures("local_resources")
+@pytest.mark.parametrize("project", targetProjects)
+@pytest.mark.assetpipeline
+@pytest.mark.SUITE_sandbox
+class TestScene_AllPlatforms(object):
+
+    @pytest.mark.BAT
+    @pytest.mark.parametrize("blackbox_param", blackbox_scene_tests, ids=blackbox_test_ids)
+    def test_SceneBlackboxTest_SourceFiles_Processed_ResultInExpectedProducts(self, workspace, ap_setup_fixture,
+                                                                              asset_processor, project, blackbox_param):
+        """
+        Please see run_fbx_test(...) for details
+        Test Steps:
+        1. Determine if blackbox is set to none
+        2. Run FBX Test
+        3. Determine if override assets is set to none
+        4. If not, re-run FBX test and validate the information in override assets
+        """
+
+        if blackbox_param is None:
+            return
+        self.run_scene_test(workspace, ap_setup_fixture, asset_processor, project, blackbox_param)
+
+        override_assets = blackbox_param.override_assets
+        # If override_assets is set, run the test again and validate the information in the override assets
+        if override_assets is not None:
+            self.run_scene_test(workspace, ap_setup_fixture,
+                                asset_processor, project, blackbox_param, True)
+
+    # Helper: Run Asset Processor with debug output enabled and Atom output disabled.
+    @staticmethod
+    def run_ap_debug_skip_atom_output(asset_processor):
+        result, output = asset_processor.batch_process(capture_output=True, extra_params=["--debugOutput",
+                                                                                          "--regset=\"/O3DE/SceneAPI/AssetImporter/SkipAtomOutput=true\""])
+        # If the test fails, it's helpful to have the output from asset processor in the logs, to track the failure down.
+        logger.info(f"Asset Processor Output: {pformat(output)}")
```

Comment:
Backlog: Use a lambda to determine if we should print the info to the logs or not.